apps/task/tests/test_apis.py

```diff
@@ -576,56 +584,43 @@ def test_weekly_summary_api_reponse_structure(self):
                 - datetime.combine(self.timeentry2.date, self.timeentry2.start_time)
         hour3 = datetime.combine(self.timeentry3.date, self.timeentry3.end_time)\
                 - datetime.combine(self.timeentry3.date, self.timeentry3.start_time)
+        hour5 = datetime.combine(self.timeentry5.date, self.timeentry5.end_time)\
```

Comment:
Rather than repeating these, create a `TimeEntry.start_datetime` and `...end_datetime` properties in the model itself. Saves time.